src/wallet/wallet.cpp

```diff
@@ -3114,7 +3115,7 @@ bool CWallet::CreateCoinStake(
         // Make sure the stake input hasn't been spent since last check
         // for now, IsSpent() requires cs_main lock due its internal call to GetDepthInMainChain.
         // This dependency will be completely removed moving forward, in #2209.
```

Comment:
we are in #2209 :) this comment can be removed.

Reply:
Done. Squashed in 4ccec74e80c924cac944279a6066aa7848ecbae9